app/frontend/wallet/shelley/shelley-crypto-provider-factory.ts

```diff
@@ -3,9 +3,13 @@ import ShelleyJsCryptoProvider from './shelley-js-crypto-provider'
 import ShelleyTrezorCryptoProvider from './shelley-trezor-crypto-provider'
 import ShelleyLedgerCryptoProvider from './shelley-ledger-crypto-provider'
 import NamedError from '../../helpers/NamedError'
+import {CryptoProvider} from '../../types'
 
 const ShelleyCryptoProviderFactory = (() => {
-  const getCryptoProvider = (cryptoProviderType, options) => {
+  const getCryptoProvider = (
+    cryptoProviderType: CRYPTO_PROVIDER_TYPES,
```

Comment:
`CryptoProviderTypeEnum` is imho a less "eyebrow raising" type name than `CRYPTO_PROVIDER_TYPES` which looks rather like a collection of something

Reply:
You are right we are not really consistent with enum naming, I noticed but I just basically copy pasted or reusused exististing enums, not making the order in them an issue of this PR, but I ll look into it :+1: